examples/setall.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+import sys
+from sys import argv
+import blinkt
+
+if len(argv)<2 or len(argv)>5:
+    sys.stderr.write( "Syntax: {0} <red> <green> <blue> [brightness]]\n".format(argv[0]) )
```

Comment:
Looks like an extra `]` has crept in here!

Reply:
Removed